src/Surfnet/Stepup/Tests/Identity/Event/EventSerializationAndDeserializationTest.php

```diff
@@ -101,7 +102,7 @@ public function an_event_should_be_the_same_after_serialization_and_deserializat
      */
     public function an_email_verification_window_should_be_the_same_after_serialization_and_deserialization()
     {
-        $window = EmailVerificationWindow::createFromTimeFrameStartingAt(TimeFrame::ofSeconds(3), DateTime::now());
+        $window = EmailVerificationWindow::createFromTimeFrameStartingAt(TimeFrame::ofSeconds(3), new DateTime(new CoreDateTime('@1000')));
```

Comment:
This is documented elsewhere, but could you reference why this is needed in your PR? Also you could refer to your other PR (and the PHP link I commented there). You never know when people need that information in the future.